[SYSTEM-1841] Add new public keys needed for Android 13.0+

c-quenin opened the pull request:
Add new public keys needed for Android 13.0+.

JIRA: [SYSTEM-1841](https://genymobile.atlassian.net/browse/SYSTEM-1841)

[SYSTEM-1841]: https://genymobile.atlassian.net/browse/SYSTEM-1841?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

c-quenin committed:
Add new public keys needed for Android 13.0+

gvercoutere-geny commented:
If we drop dev-keys usage, maybe we should at least update the README. and maybe delete dev-keys keys?

c-quenin committed:
Drop dev-keys

We don't use them anymore.
Our versions which support signed builds will be signed with these release
keys. Otherwise, they will be signed with the default AOSP test-keys.
